client/components/signup-site-preview/utils.js

```diff
@@ -0,0 +1,139 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+// Detect IE 11 and below
+export function isIE() {
+	const ua = window.navigator.userAgent;
+	return -1 !== ua.indexOf( 'MSIE' ) || -1 !== ua.indexOf( 'Trident/' );
```

Comment:
Looks like we have some code that does this in `client/lib/user-agent/index.js` - it's just a note and not a blocker though

Reply:
I did include that library first then realized I didn't need the entire lib, just a one-line test for IE.